radix-engine-common/src/math/bnum_integer.rs

```diff
@@ -59,6 +59,41 @@ macro_rules! types {
                     pub const BITS: u32 = $wrap::BITS as u32;
                     pub const BYTES: u32 = $wrap::BYTES as u32;
                     pub const N: usize = ($wrap::BYTES / 8) as usize;
+
+                    pub const fn const_from_str(from: &str) -> Result<Self, [<Parse $t Error>] > {
```

Comment:
Just wanted to make sure that you're aware of this. Not all `const fn` are pre-computed per rust documentation.